### ISSUE TITLE
[FEATURE REQUEST] Track source and build metadata

### ISSUE THREAD

**Issue by Arthurvdv** [HodorNV/ALOps#671]
![image](https://github.com/HodorNV/ALOps/assets/44637996/0de488ba-98fa-4f85-b190-3dbde406b102)

Not sure if I'm "mowing the grass under your feet", but nethertheless here i go! 

When you presenting the Business Central 2023 release wave 2 Update on Directions 4 Partners, these new properties on are going to be available in the upcoming version of Business Central. Does it make sense to have this generated by the Compile step of ALOps?

**Comment by waldo1001** [comment]
It's on the agenda (obviously) ;-).

**Comment by waldo1001** [comment]
What we're thinking of is to have one switch "on/off", and we fill in the values if it's on.  This way, we make it uniform for everyone using ALOps, and it's no work for anyone (default is "on").

Would that work for you?

**Comment by Arthurvdv** [comment]
Yeah, that should work for everybody.

Question about the build:by value, is this a seperate setting or are you retrieving this from the app.json of the App?

```json
"build" : {
   "by": "waldo, powered by ALOps"
```

**Comment by waldo1001** [comment]
This would be pushed by ALOPs - so a fixed value (something in the line of "ALOps, Hodor NV" or "ALOps on DevOps" or ... .  LIke it would be "AL-Go for GitHub"  🤷‍♂️ .. .  I think it's important to make it uniform ..

**Comment by Arthurvdv** [comment]
Ah, now I understand better what the expected value of this property needs to be, and you're right to have this as a uniform value.

**Comment by waldo1001** [comment]
Part of next release with this param:

```yaml
- task: ALOpsAppCompiler@2
  inputs:
    ...
    track_source_build_metadata: true
```

default is: true - so, the parameter should/could be used to disable it.

**Comment by AdminHodor** [comment]
Already released in version v1.460, finetuned in our latest release v1.461

Kind regards,

**Comment by Arthurvdv** [comment]
Awesome, thank you for adding this feature.
